<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="display">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="screen"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Interpolate_cb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="open_btn">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="clear_btn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="plot_label">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="plot_cb">
         <item>
          <property name="text">
           <string>Raw</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normalised</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Peaks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Desibles</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="interpolate_label">
         <property name="text">
          <string>Interpolate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="interpolate_cb">
         <item>
          <property name="text">
           <string>Cubic Spline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="peaks_chb">
         <property name="text">
          <string>Peaks</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="legend_chb">
         <property name="text">
          <string>Legend</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FI&amp;le</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionImportCSV"/>
     <addaction name="actionImportXLSX"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="actionExportCSV"/>
     <addaction name="actionExportXLSX"/>
    </widget>
    <addaction name="actionSave"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionImportCSV">
   <property name="text">
    <string>&amp;CSV</string>
   </property>
  </action>
  <action name="actionImportXLSX">
   <property name="text">
    <string>&amp;XLSX</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="text">
    <string>&amp;CSV</string>
   </property>
  </action>
  <action name="actionExportXLSX">
   <property name="text">
    <string>&amp;XLSX</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
